README.md

```diff
@@ -97,9 +104,16 @@ Otherwise, the following steps will work for any language:
 
 ## Detailed usage
 
-### Enhancing streaming video with asbplayer features
+First, install the Chrome [extension](https://chromewebstore.google.com/detail/asbplayer-language-learni/hkledmpjpaehamkiehglnbelcpdflcab).

```

Comment:
I don't think this necessarily makes sense directly under `Detailed usage` since there are plenty of users who don't install the extension to use asbplayer with local files.